src/OpenFeature/Api.cs

```diff
@@ -104,12 +104,12 @@ public FeatureProvider GetProvider(string clientName)
         public Metadata? GetProviderMetadata() => this.GetProvider().GetMetadata();
 
         /// <summary>
-        /// Gets providers metadata assigned to the given clientName. If the clientName has no provider
+        /// Gets providers metadata assigned to the given domain. If the domain has no provider
         /// assigned to it the default provider will be returned
         /// </summary>
-        /// <param name="clientName">Name of client</param>
+        /// <param name="domain">Name of client</param>

```

Comment:
Same as https://github.com/open-feature/dotnet-sdk/pull/294/files#r1715020674

```suggestion
        /// <param name="domain">An identifier which logically binds clients with providers</param>

```